fast_api/routes/organization.py

```diff
@@ -281,23 +283,37 @@ def get_mapped_sorted_paginated_users(
                 user.last_interaction.isoformat() if user.last_interaction else None
             ),
             "role": user.role,
-            "organizationName": (
+            "organization": (
                 organization_manager.get_organization_by_id(str(user.organization_id))[
                     "name"
                 ]
                 if user.organization_id
                 else ""
             ),
+            "email": user.email,
+            "verified": user.verified,
+            "created_at": user.created_at.isoformat(),
+            "metadata_public": user.metadata_public,
+            "sso_provider": user.sso_provider,
         }
         for user in active_users
     ]
-    active_users = {user["id"]: user for user in active_users}
 
-    data, final_len = user_manager.get_mapped_sorted_paginated_users(
-        active_users, body.sort_key, body.sort_direction, body.offset, body.limit
+    final_users = []
+    final_users = sorted(
+        active_users,
+        key=lambda x: (x[body.sort_key] is None, x.get(body.sort_key, "")),
+        reverse=body.sort_direction == -1,
```

Comment:
the sort & pagination should be done on the database for the fields that exist on the database. Less data we need to extend etc. could also already provide the org name so we dont need to worry about it here. 

I can help with that :)